relayer/src/services/relayer/optimism.ts

```diff
@@ -76,6 +76,35 @@ export class OptimismRelayerService extends BaseRelayerService {
     return;
   }
 
+  async safeRelayMessage(message: CrossChainMessage): Promise<boolean> {
+    try {
+      // Finalize message via OptimismPortal (it calls the L1CrossDomainMessenger internally)
+      (await this.messenger.finalizeMessage(message)).wait(1);
```

Comment:
`wait(1)` means we wait max 1 second here? Is that enough? What happens if the operation takes longer than that?